packages/pwa-kit-runtime/src/utils/ssr-proxying.js

```diff
@@ -683,11 +683,18 @@ export const rewriteProxyResponseHeaders = ({
  * @private
  * @type {string[]}
  */
-export const X_HEADERS_TO_REMOVE = [
+export const X_HEADERS_TO_REMOVE_PROXY = ['x-mobify-access-key']
+
+/**
+ * List of x- headers that are removed from origin requests.
+ * @private
+ * @type {string[]}
+ */
+export const X_HEADERS_TO_REMOVE_ORIGIN = [
```

Comment:
@kieran-sf Would this be considered a breaking change?

If customers were relying on the removal or existence of those headers and suddenly found them present or missing in the requests, Could this lead to unexpected behaviour, errors or data exposure?

At the very least, are we planning on documenting the changes that could impact customers and provide alternative methods for those who need to maintain the current behaviour?

Reply:
We reviewed the changes and agreed that although there is technically a change in the behavior of the proxies, we don't expect significant consequences for existing apps that now include the headers that we were previously removing.